superset/security/manager.py

```diff
@@ -784,11 +786,17 @@ def sync_role_definitions(self) -> None:
 
         self.create_custom_permissions()
 
+        #Fetch all pvms
+        pvms = self.get_session.query(PermissionView).options(
+            eagerload(PermissionView.permission),
+            eagerload(PermissionView.view_menu)).all()
+        pvms = [p for p in pvms if p.permission and p.view_menu]
```

Comment:
Does this check do anything? These are required relations, aren't they?

Reply:
Yes this is required relation but as this condition was written in previous query I thought of not removing it.

Reply:
Ah yeah, I see you haven't changed this. Fair enough.